x-pack/test_serverless/functional/test_suites/common/management/data_views/_data_view_create_delete.ts

```diff
@@ -31,7 +29,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await esArchiver.loadIfNeeded(
         'x-pack/test_serverless/functional/es_archives/kibana_sample_data_flights_index_pattern'
       );
+      await kibanaServer.importExport.load(
```

Comment:
Question: if this fixes the MKI error, isn't L29 + L42 redundant? 
@pheyos wrote in the issue 

> The root cause is in both cases direct system indices access. AFAICS, this is happening when loading / unloading the archive x-pack/test_serverless/functional/es_archives/kibana_sample_data_flights_index_pattern. This kind of access is not available in MKI and the tests need to be adjusted (e.g. by loading things through kibanaServer.importExport or other APIs).

Since those functions are still here, should it work? will it work when they are removed?

Reply:
@kertal line 29 is importing regular ES docs while line 32 is importing saved objects. I'll defer to @pheyos as to whether I'm doing this correctly.

Reply:
Yes, that's correct @mattkime. The other issue was reported quite a while back and I suspect some other things have changed as well in the mean time. I ran the tests against an observability MKI project and they passed.